src/controllers/expense.controller.js

```diff
@@ -0,0 +1,97 @@
+const expenseService = require('../services/expense.service.js');
+const userService = require('../services/user.service.js');
+
+const get = (req, res) => {
+  const { categories, userId, from, to } = req.query;
+
+  res.send(expenseService.getAll(categories, userId, from, to));
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+
+  const expense = expenseService.getById(+id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = (req, res) => {
+  const { userId, title, amount, category, note, spentAt } = req.body;
+  const existedUser = userService.getById(userId);
+
+  if (
+    !userId ||
+    !title ||
+    !amount ||
+    !category ||
+    !note ||
+    !spentAt ||
+    !existedUser
```

Comment:
The validation here checks for falsy values, which might incorrectly reject valid inputs like `0` or an empty string. Consider using a more robust validation method to ensure all required fields are correctly validated.